api/src/server.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Module dependencies.
+ */
+const debug = require("debug")("vudlprepjs:server"); // eslint-disable-line
```

Comment:
Any idea why eslint needs to be disabled here? Is there a reason we can't change const to import to make it happy (if that's what it's complaining about)?

Reply:
Made Change